experiment/resultstore/convert.go

```diff
@@ -124,6 +132,32 @@ func convert(project, details string, url gcs.Path, result downloadResult) (resu
 				URL:         buildLog, // ensure build-log.txt appears as the invocation log
 			},
 		},
+		Properties: []resultstore.Property{
+			{
+				Key:   "Job Name",
+				Value: jobName,
+			},
+			{
+				Key:   "Started Timestamp",
+				Value: strconv.FormatInt(started.Timestamp, 10),
+			},
+			{
+				Key:   "Node",
+				Value: started.Node,
+			},
+			{
+				Key:   "Pull",
+				Value: started.Pull,
```

Comment:
This is not always set and won't be useful without the repo, right?

Reply:
Thanks Steve. Yes, you are correct. Though at the moment, for the experiment, I think it is acceptable to have an empty value for the key. I will add a comment beside it to note this value and the started.Node value can be empty.